en/developer_guide/navigation_panel.rst

```diff
@@ -41,14 +41,53 @@ The menu link must have the following attributes :
 * id : merge of the "nav-" string and the name of the strategy as returned by
   ``NavigationPanelInterface::getName()``, for instance "nav-user"
 
-It can also have data-* attributes that can help you to define custom properties to be used in a Backbone route,
-for example :
+Menu and data table
+~~~~~~~~~~~~~~~~~~~
+
+Many panel entries allow to consult records formated in a data table.
+This data table could be configured : name of the column, visible or not by default, type of search field...
+As this view is based on `DataTables library`_, all possible options in column definition are allowed.
+Here is an example of yml configuration:
+
+.. code-block:: yaml
+
+    open_orchestra_backoffice.navigation_panel.trashcan.parameters :
+        -
+          name : name
+          title : open_orchestra_backoffice.table.trashcan.name
+          visible : true
+          activateColvis : true
+          searchField : text
+
+The parameter ``open_orchestra_backoffice.navigation_panel.trashcan.parameters`` is injected to panel strategy whith a translator which implements ``Symfony\Component\Translation\TranslatorInterface``.
+The translator try to translate each associated value in current context, in this case ``open_orchestra_backoffice.table.trashcan.name``.
```

Comment:
The Symfony translator will try to translate each value, in this case ``...`` will be translated